training/utils/upload_manager.py

```diff
@@ -217,7 +217,7 @@ def main():
     parser.add_argument('directories', nargs='+', help='Directories to upload')
 
     args = parser.parse_args()
-    aws_process_args(args)
+    aws_process_args(args, required=True)
```

Comment:
should this be hardcoded to True or could we pass `--aws-enabled` or something to set this to true? Seems like this'll just error out by default

Reply:
This line is in the CLI portion of the script. This will not impact the training script.